Remove paging when we're filtering the metadata type

acron0 opened the pull request:
**Remove paging when we're filtering the metadata type**
This is a compromise solution to working around the problem of Datastore
not filtering for us (which breaks paging). We will re-look at this
problem once search is available.

jasebell commented on start-nginx.sh:
```diff
@@ -17,42 +17,49 @@ server {
 
             proxy_pass http://${SERVER_ADDR}:${SERVER_PORT};
 
+
```
Three blank lines needed?  Apols for the nitpick.

elisehuard replied:
not really, true, quick to fix

jasebell commented:
Small insignificant nitpick but otherwise it's a :shipit: from me.

Commit:
Remove paging when we're filtering the metadata type

This is a compromise solution to working around the problem of Datastore
not filtering for us (which breaks paging). We will re-look at this
problem once search is available.
